src/main/resources/database_dump.sql

```diff
@@ -169,6 +169,7 @@ CREATE TABLE `member` (
   `nickname` varchar(255) DEFAULT NULL,
   `provider` varchar(10) NOT NULL,
   `organization` varchar(255) DEFAULT NULL
+UNIQUE KEY `unq_member` (`email`,`provider`)
```

Comment:
You need to add a comma at the end of the 171 line. 
Please be careful about indentation.

```suggestion
  `organization` varchar(255) DEFAULT NULL,
  UNIQUE KEY `unq_member` (`email`,`provider`)
```